Tests/SentryTests/Networking/RateLimits/SentryConcurrentRateLimitsDictionaryTests.swift

```diff
@@ -33,8 +33,8 @@ class SentryConcurrentRateLimitsDictionaryTests: XCTestCase {
     }
 
     func testConcurrentReadWrite() {
-        let queue1 = DispatchQueue(label: "SentryConcurrentRateLimitsStorageTests1", qos: .background, attributes: [.concurrent, .initiallyInactive])
-        let queue2 = DispatchQueue(label: "SentryConcurrentRateLimitsStorageTests2", qos: .utility, attributes: [.concurrent, .initiallyInactive])
+        let queue1 = DispatchQueue(label: "SentryConcurrentRateLimitsStorageTests1", attributes: [.concurrent, .initiallyInactive])
```

Comment:
`m`: I believe we should have two queues with different priorities, rather than the same one.

Reply:
Thanks for double-checking. I looked at the test and I don't think different priorities are required to check if the implementation is thread safe.